Makefile

```diff
@@ -104,7 +104,7 @@ LIB_LDFLAGS        = $(COMMON_LDFLAGS) $(LDFLAGS_$@) $(LDFLAGS_lib) -Wl,--no-und
 TEST_LDFLAGS       = $(COMMON_LDFLAGS) -L$(objdir)/libtraceevent -ltraceevent
 
 ifeq ($(DEBUG), 1)
-  COMMON_CFLAGS += -O0 -g3 -DDEBUG_MODE=1
+  COMMON_CFLAGS += -O0 -g3 -DDEBUG_MODE=1 -Werror=strict-prototypes
```

Comment:
Please do it in a separate commit.  Also I think we can add `-Werror` here and add `-Wstrict-prototypes` to all conditions.

Reply:
OK, Thanks for code review!